soul-spring-boot-starter/soul-spring-boot-starter-sync-data-center/soul-spring-boot-starter-sync-data-http/pom.xml

```diff
@@ -38,5 +38,22 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-test</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>com.squareup.okhttp3</groupId>
+            <artifactId>okhttp</artifactId>
+            <version>3.11.0</version>
```

Comment:
okhttp's version has been declared in root pom.xml

Reply:
![image](https://user-images.githubusercontent.com/13709330/103173611-0cfb7600-4897-11eb-8f4a-f436604d7169.png)
The declared version of okhttp leads to building failure.

Should I refactor these tests with alternatives?

Reply:
Yes. you can try `WireMockRule`. There is a example [here](https://github.com/dromara/soul/blob/master/soul-client/soul-client-common/src/test/java/org/dromara/soul/client/common/utils/OkHttpToolsTest.java).

Reply:
Thx a lot, I'll fix it.